pom.xml

```diff
@@ -64,7 +64,7 @@
     <properties>
         <revision>2.22</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.130</jenkins.version>
```

Comment:
is the jenkins bump actually needed?

Reply:
i recall needing to do that to make my local testing work, i will take a look and see if it is really needed outside that context.